README.md

```diff
@@ -144,14 +144,13 @@ For example:
               value: "1"
 ```
 
-#### Set an Update Time Window
+#### Set an Update Time Window - DEPRECATED
+Note!! that these settings are deprecated and will be removed in a future release. You should use the Scheduler settings below.  If you still decide to use these settings, please use "hour:00:00" format only instead of "HH:MM:SS".
```

Comment:
Match the code style used elsewhere in this document.
```suggestion
#### Set an Update Time Window - DEPRECATED

Note!! that these settings are deprecated and will be removed in a future release.
You should use the Scheduler settings below.
If you still decide to use these settings, please use "hour:00:00" format only instead of "HH:MM:SS".
```